tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderSummarySteps.cs

```diff
@@ -66,7 +66,18 @@ public async Task ThenTheOrderSummarySectionsHaveTheFollowingValues(Table table)
 
             actual.Sections.Should().BeEquivalentTo(expected);
         }
-        
+
+        [Then(@"the order Section Status is (.*)")]
+        public async Task ThenTheOrderSummarySectionStatusIs(string expectedStatus)
```

Comment:
Very minor suggestion to match the name of the method with the step description.